angular-locale_es-us.js

```diff
@@ -119,7 +119,7 @@ $provide.value("$locale", {
     ]
   },
   "id": "es-us",
-  "localeID": "es_US",
+  "localeID": "es-US",
```

Comment:
Why was this changed to - from _? and why are the rest of these files - rather than _? I don't believe we use this property but we should keep it consistent with the other files